sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Generated/Attachments.cs

```diff
@@ -433,7 +433,7 @@ public virtual AsyncPageable<BinaryData> GetAttachmentsByPartyIdAsync(string par
         /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
         /// <returns> The <see cref="Pageable{T}"/> from the service containing a list of <see cref="BinaryData"/> objects. Details of the body schema for each item in the collection are in the Remarks section below. </returns>
         /// <include file="Docs/Attachments.xml" path="doc/members/member[@name='GetAttachmentsByPartyId(string,IEnumerable{string},IEnumerable{string},IEnumerable{string},IEnumerable{string},IEnumerable{string},IEnumerable{string},DateTimeOffset?,DateTimeOffset?,DateTimeOffset?,DateTimeOffset?,int?,string,RequestContext)']/*" />
-        public virtual Pageable<BinaryData> GetAttachmentsByPartyId(string partyId, IEnumerable<string> resourceIds = null, IEnumerable<string> resourceTypes = null, IEnumerable<string> ids = null, IEnumerable<string> names = null, IEnumerable<string> propertyFilters = null, IEnumerable<string> statuses = null, DateTimeOffset? minCreatedDateTime = null, DateTimeOffset? maxCreatedDateTime = null, DateTimeOffset? minLastModifiedDateTime = null, DateTimeOffset? maxLastModifiedDateTime = null, int? maxPageSize = null, string skipToken = null, RequestContext context = null)
```

Comment:
I think we should consider either hand writing these as option bags or creating a config which takes a list of methods to convert into options bags for swagger use case.

In swagger we won't have the client.tsp capabilities and since we don't want an automatic decision to convert to options bag based on threshold a configuration would probably be a good thing.

Reply:
We currently decide not implement the option bag.